src/beautifiers/prettier.coffee

```diff
@@ -0,0 +1,46 @@
+"use strict"
+
+Beautifier = require('./beautifier')
+prettier = require("prettier")
+
+module.exports = class Prettier extends Beautifier
+  name: "Prettier"
+  link: "https://github.com/prettier/prettier"
+  options: {
+    _:
+      tabWidth: "indent_size"
+      useTabs: ["indent_with_tabs", "indent_char", (indent_with_tabs, indent_char) ->
+        return (indent_with_tabs is true) or (indent_char is "\t")
+      ]
+    JavaScript:
+      bracketSpacing: "object_curly_spacing"
+    TypeScript: false
+    CSS: false
+    LESS: false
+    SCSS: false
+    Vue: false
+    JSON: false
+    Markdown: false
+  }
+
+  beautify: (text, language, options) ->
+    return new @Promise((resolve, reject) ->
+      _ = require('lodash')
+
+      prettierLanguage = _.find(prettier.getSupportInfo().languages, 'name': language)
+      if prettierLanguage
+        parser = prettierLanguage.parsers[0]
+      else
+        reject(new Error("Unknown language for Prettier"))
+
+      filePath = atom.workspace.getActiveTextEditor().getPath()
```

Comment:
Use `context.filePath` instead.
See https://github.com/Glavin001/atom-beautify/blob/788d002402faa782d3cf01dea7517c238daf0cbb/src/beautifiers/rustfmt.coffee#L20-L21 for example.

Reply:
What does Prettier `resolveConfig` do if `filePath = undefined`?

Reply:
It returns null if it can't find anything.